datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -103,6 +104,13 @@ def run_check(
 
         return self.exec_command(command, capture=capture, interactive=break_point is not None)
 
+    def wait_agent_ready(self):
+        for _ in range(10):
+            result = self.exec_command('ls /etc/datadog-agent/datadog.yaml', capture=True)
```

Comment:
Why not using something like `file_exist` from utils? https://github.com/DataDog/integrations-core/blob/1f59a18a72da4f0c1e689e15720ee0d51119fe3d/datadog_checks_dev/datadog_checks/dev/utils.py#L92

Reply:
It runs in the docker env.